src/http/controllers/device.js

```diff
@@ -0,0 +1,33 @@
+import deviceService from '../../services/verify';
+import deviceValidation from '../validations/verifyDevice';
+
+const verifyDevice = async (req, res) => {
+  const { deviceId } = req.body;
+  const verification = await deviceService.verify(deviceId);
+  if (verification.status === 201) {
```

Comment:
I already left a comment somewhere about the status code of successful device verification. It should be a `204` not `201`. `201` implies that a resource is created, we are not creating any resource here.